pages/docs.json

```diff
@@ -93,6 +93,12 @@
         },
         {
           "title": "Test Utilities"
+        },
+        {
+          "title": "Stylelint Processor"
+        },
+        {
+          "title": "Stylelint Config"
```

Comment:
Rather than making these two separate sections let's merge them into one section called "Stylelint"? Now that they aren't two READMEs anymore it doesn't make a lot of sense for them to be separate, right? Especially if we do #88.

Reply:
Agreed! :wink: